_includes/accessibility-tests/wcag-details.html

```diff
@@ -152,6 +152,11 @@
   {% assign criterion_name = "Focus Not Obscured (Minimum) (WCAG 2.2)" %}
   {% assign criterion_url = "https://www.w3.org/TR/WCAG22/#focus-not-obscured-minimum" %}
 {% endif %}
+{% if item.wcag_criterion == "2.5.7" %}
+  {% assign criterion_level = "AA" %}
+  {% assign criterion_name = "Dragging Movements" %}
+  {% assign criterion_url = "https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html" %}
```

Comment:
@RachelCorsino 

**issue:** We should follow the existing model for these 2.2 links

```suggestion
  {% assign criterion_url = "https://www.w3.org/TR/WCAG22/#dragging-movements" %}
```

Reply:
Resolved in [23c28b1](https://github.com/uswds/uswds-site/pull/2963/commits/23c28b1b772ee9faf1bc5fb99040b3d3a7fa4f45)